src/libraries/System.Runtime/tests/System.Globalization.Calendars.Tests/ISOWeek/ISOWeekTests.cs

```diff
@@ -187,6 +244,25 @@ public DateData(DateTime date, int year, int week, DayOfWeek dayOfWeek)
             }
         }
 
+        private struct DateData
```

Comment:
Isn't already defined above?

Reply:
Apologies, this was a diff error in the last commit. It should be fixed now.

Reply:
No worries at all. Thanks for fixing it!